### ISSUE TITLE
Where to find the demo data (input.fasta) used in fusion transcript finding (https://github.com/Magdoll/cDNA_Cupcake/wiki/Best-practice-for-fusion-transcript-finding)

### ISSUE THREAD

**Issue by wzhang42** [Magdoll/cDNA_Cupcake#224]
Hi, Magdoll,   Thanks you so much for your detailed description about Best practice for fusion transcript finding . I am now playing with the commands/scripts shared in https://github.com/Magdoll/cDNA_Cupcake/wiki/Best-practice-for-fusion-transcript-finding .    I am trying to replicate your result step by step, but where to find the demo data as a input.fasta?
Thank you in advance.

**Comment by Magdoll** [comment]
Hi @wzhang42 , 
I did not put out a demo dataset, but we have an old MCF-7 Iso-Seq dataset where I used `fusion_finder.py` to identify fusion candidates.
http://datasets.pacb.com.s3.amazonaws.com/2015/IsoSeqHumanMCF7Transcriptome/list.html

Specifically you can download the [fusion fasta](http://datasets.pacb.com.s3.amazonaws.com/2015/IsoSeqHumanMCF7Transcriptome/IsoSeq_MCF7_2015edition_polished.fusion.fasta) file to use that to test.

Thanks!
-Liz

**Comment by wzhang42** [comment]
Hi, Magdoll, 
   Many thanks for your reply. I have download the  [fusion fasta](http://datasets.pacb.com.s3.amazonaws.com/2015/IsoSeqHumanMCF7Transcriptome/IsoSeq_MCF7_2015edition_polished.fusion.fasta) file and used the same commands and parameters at (https://github.com/Magdoll/cDNA_Cupcake/wiki/Best-practice-for-fusion-transcript-finding) 1) minimap2 against hg38 & sort 2.)   fusion_finder.py , 3.) SQANTI (gencode.v33) , 4.) fusion_collate_info.py.  However, in the final annotation result files (output.fusion.annotated.txt + output.fusion.annotated_ignored.txt, or the output.fusion.gff), I could not find the mentioned PBfusion.142 with the break point at the close position at chr1:6825211. (Please you correct me if I used the wrong analyzing procedure)
I also adopted the close procedure and run several rounds to my own data, and generated several PBfusion candidates. However, we do not know whether they are just false positive (many of them are mapped to intron regions).        
I am also puzzled to configure the optimal parameters for fusion_finder.py (  -c as 0.05 for per locus coverage , -t as 0.98 for total coverage, -d as 10K for distance) . 
 In short, I do not know how to configure the correct /optimal parameter  and also do not know the procedure to validate the true positive PB fusion events.  
Sorry for many questions. 
Many thanks again.
Wenchao
